src/app/containers/Ad/Amp/index.jsx

```diff
@@ -103,7 +103,7 @@ const AMP_ACCESS_DATA = endpoint => ({
 const LABEL_LINK = 'https://www.bbc.com/usingthebbc/cookies/';
 
 export const AMP_ACCESS_FETCH = service => {
-  const togglesEndpoint = `${process.env.SIMORGH_TOGGLES_URL}/toggles?application=simorgh&service=${service}`;
+  const togglesEndpoint = `${process.env.SIMORGH_TOGGLES_URL}?application=simorgh&service=${service}`;
```

Comment:
Does this need to follow the same changes as above as well?
```suggestion
  const configEndpoint = `${process.env.SIMORGH_CONFIG_URL}?application=simorgh&service=${service}`;
```

Reply:
Thanks for spotting these changes @lokh01 . I've added these